R/simulation.R

```diff
@@ -49,6 +47,11 @@ Simulation <- R6::R6Class(
     initialize = function(ref, sourceFile = NULL) {
       super$initialize(ref)
       private$.sourceFile <- sourceFile
+      private$.buildConfiguration <- rClr::clrGet(self$ref, "BuildConfiguration")

```

Comment:
This is just an optimization to avoid creating .net object all the time. Just saving a reference to build config and settings and I can use them throuhgout